package/com.unity.formats.usd/Dependencies/USD.NET/serialization/AccessMask.cs

```diff
@@ -28,6 +58,6 @@ namespace USD.NET
     /// </remarks>
     public class AccessMask
     {
-        public Dictionary<SdfPath, HashSet<MemberInfo>> Included = new Dictionary<SdfPath, HashSet<MemberInfo>>();
+        public Dictionary<SdfPath, DeserializationContext> Included = new Dictionary<SdfPath, DeserializationContext>();
```

Comment:
This field is named `Included`, does the serialization context belong here, or as a sibling to this field?

`AccessMask` is also no longer a great name for this - it should be something more like, `PrimStreamingContextMap`, or something.

We could make these changes in a future PR, but maybe add a comment?